README.md

```diff
@@ -1,32 +1,48 @@
+| pylint.cwd              | `${workspaceFolder}`                                                                                                                   | Sets the current working directory used to lint Python files with Pylint. By default, it uses the root directory of the workspace `${workspaceFolder}`. You can set it to `${fileDirname}` to use the parent folder of the file being linted as the working directory for Pylint.                                                                                                                                                                                                                                                                                                          |
+| pylint.severity         | `{ "convention": "Information", "error": "Error", "fatal": "Error", "refactor": "Hint", "warning": "Warning", "info": "Information" }` | Mapping of Pylint's message types to VS Code's diagnostic severity levels as displayed in the Problems window. You can also use it to override specific Pylint error codes. E.g. `{ "convention": "Information", "error": "Error", "fatal": "Error", "refactor": "Hint", "warning": "Warning", "W0611": "Error", "undefined-variable": "Warning" }`                                                                                                                                                                                                                                        |
+| pylint.path             | `[]`                                                                                                                                   | Path or command to be used by the extension to lint Python files with Pylint. Accepts an array of a single or multiple strings. If passing a command, each argument should be provided as a separate string in the array. Note: Using this option may slowdown linting. It also takes precedence over the the `pylint.importStrategy` setting. <br>Examples: <br>- `["~/global_env/pylint"]` <br>- `["conda", "run", "-n", "lint_env", "python", "-m", "pylint"]`                                                                                                                                                                                                 |
```

Comment:
We need to let users know that, setting this to `pylint` will pick it from PATH (this was a feature request for linters).

Reply:
Added a note to clarify that

Reply:
Just to double check, it should be set to `["pylint"]` right? and not `"pylint"`?

Reply:
Correct, it is `["pylint"]`